sui/deploy-contract.js

```diff
@@ -0,0 +1,76 @@
+const { saveConfig, printInfo } = require('../evm/utils');
+const { Command, Argument } = require('commander');
+const { publishPackage, updateMoveToml } = require('@axelar-network/axelar-cgp-sui/scripts/publish-package');
+
+const { addBaseOptions } = require('./cli-utils');
+const { getWallet, printWalletInfo } = require('./sign-utils');
+const { loadSuiConfig, findPublishedObject } = require('./utils');
+
+// Add more contracts here to support more modules deployment
+const contractMap = {
+    gasService: {
+        packageName: 'gas_service',
+        contractName: 'GasService',
+        displayName: 'Gas Service',
```

Comment:
not really needed, `GasService` is fine. If we do need display name in the future, it should go in the main config